### PR TITLE
draggable clicking on mobile devices fixed 

### PR DESCRIPTION
**Pull request by saadiqbal-dev**

### CODE REVIEW

File: src/components/MobileCore.js [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+`;
+
+const MobileCore = (props) => {
+  const account = useAccount();
+  const location = useLocation();
+
+  const [showPretendModal, setShowPretendModal] = useState(false);
+
+  return (
+    // <Draggable position={null} axis="y" bounds="parent">
```
Comment (elliotBraem) [inline src/components/MobileCore.js]: So we disable draggable on Mobile?

File: src/App.js [hunk 2]
```diff
@@ -171,11 +172,13 @@ function App() {
               <BosLoaderBanner />
               <FeedPage {...passProps} />
               <Core {...passProps} />
+              {/* <MobileCore {...passProps} /> */}
```
Comment (elliotBraem) [inline src/App.js]: Does MobileCore get used?

File: src/components/MobileCore.js (hunk 1 above)
Comment (saadiqbal-dev) [reply src/components/MobileCore.js]: No, it works on mobile too

File: src/App.js (hunk 2 above)
Comment (saadiqbal-dev) [reply src/App.js]: No, it isn't used, i removed the extra code

**Pull request merged**